docs/dev/contribute.rst

```diff
@@ -12,21 +12,16 @@ Be careful, make sure the branch you are working on does **not** start with
 0. What and where to add
 ------------------------
 
-* New **test structures** (alignment marks, vdp...) and new **geometries**
-  (hyper taper...) go to ``geometries``.
-* New **devices** (ntron, htron, snspd...) go to ``devices``.
-* New **circuits** (logic gates, counter, any kinf of circuit made of
-  devices...) go to ``circuits``.
-* New **design tools** (functions useful for building designs) go to
-  ``utilities``.
-* New **cells** (made of circuits integrated with utilities tools) go to
-  ``design``. 
+Before contributing to the package, make sure its structure and organization are
+clear to you. The contributions should go in the same direction as how the
+package was though in terms of hierarchy. If you are already a user of the
+package, the contribution should be quite straigh forward; otherwise, please
```

Comment:
```diff
-clear to you. The contributions should go in the same direction as how the
-package was though in terms of hierarchy. If you are already a user of the
-package, the contribution should be quite straigh forward; otherwise, please
+clear to you. The contributions should follow the intentions of the package's
+hierarchy/organization. If you are already a user of the package, the
+contribution should be quite straightforward; otherwise, please
```